core/data-test/src/main/kotlin/com/google/samples/apps/nowinandroid/core/data/test/DefaultZoneIdTimeZoneMonitor.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid.core.data.test
+
+import com.google.samples.apps.nowinandroid.core.data.util.TimeZoneMonitor
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.datetime.TimeZone
+import javax.inject.Inject
+
+class DefaultZoneIdTimeZoneMonitor @Inject constructor() : TimeZoneMonitor {
+    override val currentTimeZone: Flow<TimeZone> = flowOf(TimeZone.of("Europe/Warsaw"))
```

Comment:
I'm wondering, shouldn't we use the system default instead of an hardcoded value?
```kotlin
TimeZone.currentSystemDefault()
```

Same question for `TestTimeZoneMonitor.kt`.

Reply:
Using a fixed time-zone makes sense to me for reproducability - we previously ran into an issue where screenshot tests could fail due to using the local time zone instead of a fixed time zone: https://github.com/android/nowinandroid/pull/876#pullrequestreview-1587229309

Reply:
That makes sense, thanks for the link :)

Reply:
Thanks for verifying!